device-types/Cisco/WS-C3750X-12S.yaml

```diff
@@ -35,7 +35,7 @@ interfaces:
     type: 1000base-x-sfp
 module-bays:
   - name: Network Module
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than NM, as interfaces on the network module will populate interface names with '1' which should follow how the switch shows the interfaces for that network module.